notebooks/official/stable_diffusion/dreambooth_stablediffusion.ipynb

```diff
@@ -0,0 +1,1326 @@
+{
```

Comment:
<p>fp16=False ?  Hum, but you trained as mixed precision?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1491/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>

Reply:
Good catch!